### ISSUE TITLE
Uncaught TypeError: this.editor.getCursors is not a function

### ISSUE THREAD

**Issue by alaarezk1989** [Freyskeyd/atom-alignment#79]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 15.04
**Thrown From**: [atom-alignment](https://github.com/Freyskeyd/atom-alignment) package 0.13.0

### Stack Trace

Uncaught TypeError: this.editor.getCursors is not a function

```
At /home/ubunto/.atom/packages/atom-alignment/lib/aligner.coffee:16

TypeError: this.editor.getCursors is not a function
    at Aligner.module.exports.Aligner.__getRows (/packages/atom-alignment/lib/aligner.coffee:16:40)
    at Aligner.__getRows (/packages/atom-alignment/lib/aligner.coffee:1:1)
    at Aligner.module.exports.Aligner.align (/packages/atom-alignment/lib/aligner.coffee:218:14)
    at Aligner.align (/packages/atom-alignment/lib/aligner.coffee:1:1)
    at alignLinesMultiple (/packages/atom-alignment/lib/atom-alignment.coffee:48:7)
    at /packages/atom-alignment/lib/atom-alignment.coffee:33:17)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/usr/share/atom/resources/app/src/atom-environment.js:1314:34)
    at EventEmitter.outerCallback (/usr/share/atom/resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -8:18.3.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
  8x -7:55 core:backspace (input.hidden-input)
     -7:23.2.0 intentions:highlight (input.hidden-input)
     -7:22.2.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -7:18.3.0 intentions:highlight (input.hidden-input)
     -7:17.6.0 command-palette:toggle (input.hidden-input)
     -7:09.1.0 atom-alignment:align (input.hidden-input)
     -7:06.5.0 intentions:highlight (input.hidden-input)
  4x -7:02.6.0 editor:unfold-current-row (input.hidden-input)
  5x -6:59.3.0 atom-alignment:alignMultiple (input.hidden-input)
  4x -2:30.5.0 intentions:highlight (input.hidden-input)
     -0:32.3.0 atom-alignment:alignMultiple (input.hidden-input)
     -0:30.4.0 intentions:highlight (input.hidden-input)
     -0:29.5.0 core:select-all (input.hidden-input)
     -0:26.5.0 intentions:highlight (input.hidden-input)
  5x -0:25.6.0 atom-alignment:alignMultiple (input.hidden-input)
```

### Non-Core Packages

```
alignment 1.1.5 
atom-alignment 0.13.0 
atom-php-navigation 0.6.0 
busy-signal 1.4.3 
define-jump 1.0.1 
goto-definition 1.3.4 
hyperclick 0.1.2 
intentions 1.1.2 
js-hyperclick 1.11.0 
linter 2.2.0 
linter-ui-default 1.6.2 
```

**Comment by caleb531** [comment]
Hi @alaarezk1989,

I recently encountered this error, and have submitted a PR to fix it. Please check out my fork at https://github.com/caleb531/atom-alignment if you want to get the fix.

Caleb

**Comment by Freyskeyd** [comment]
fixed by #83 
will be add to next release
